recipes/dlpoly-py/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "dlpoly-py" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 80ec23a140bd0f8f52c3dabdac7e986da1c843bdcc689f03893350859aa69ef6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - sphinx
+  run:
+    - python >=3.6
+    - numpy >=1.5.0
+    - matplotlib >=2.0.0
+    - flake8
+    - ruamel.yaml
+    - coverage
+    - pytest
+    - pytest-cov
+    - tox
+    - sphinx
+
+test:
+  imports:
+    - numpy
```

Comment:
You don't need to test the import of `numpy`

Reply:
@synapticarbors thanks for the catch. I corrected the metafile.